README.md

```diff
@@ -1,13 +1,11 @@
-# Project Auth
-
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+<!-- @format -->
 
-## The problem
+# Project Auth
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+In this project we have build an API with authentication to implement a registration flow. The frontend consists of a form to sign up and sign in. Once the user is logged in you can access the restricted content.
 
 ## View it live
 
-Every project should be deployed somewhere. Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+FE: https://vigorous-gates-2c453b.netlify.app/signin
```

Comment:
To fix it you need to create `_redirects` file in the `public` directory of your react project and paste it inside:
``` 
/* /index.html 200
```


![image](https://user-images.githubusercontent.com/50926159/120919420-ca498f80-c6b9-11eb-8774-4cb452a25e21.png)